packages/@react-spectrum/button/src/ActionButton.tsx

```diff
@@ -53,9 +50,7 @@ function ActionButton(props: SpectrumActionButtonProps, ref: FocusableRef) {
             'spectrum-ActionButton',
             {
               'spectrum-ActionButton--quiet': isQuiet,
-              'spectrum-ActionButton--emphasized': isEmphasized,
               'is-active': isPressed,
-              'is-selected': isSelected,
```

Comment:
Removing selected state from ActionButton for now since it's not really a proper toggle button. Spectrum is considering making a separate ToggleButton component in the future.